Common/Sources/View/Error/ErrorView.swift

```diff
@@ -111,4 +114,18 @@ public final class ErrorView {
                                                 message: "network_connection_error_message".localized(),
                                                 handler)
     }
+
+    private func authenticationError(_ error: HyperwalletErrorType) {
+        HyperwalletUtilViews.showAlert(viewController,
+                                       title: "authentication_error_title".localized(),
+                                       message: error.getAuthenticationError()?.message() ??
+                                        "authentication_error_message".localized(),
+                                       actions: UIAlertAction.close({ (_) in
+                                        NotificationCenter.default
```

Comment:
Update the indent as below.
<img width="645" alt="Screenshot 2020-03-23 at 3 59 44 PM" src="https://user-images.githubusercontent.com/55200187/77307501-6094ce00-6d1f-11ea-99b5-c90b1cd0bde4.png">

Please reindent the whole class.

Reply:
Indent added